### ISSUE TITLE
Broken Background after optimization

### ISSUE THREAD

**Issue by LeJared** [jakubpawlowicz/clean-css#629]
This Rule
```css
.test {
  background: #ffffff none repeat scroll 0 0;
}
```
gets optimized to
```css
.test{background:none}
```
expected result
```css
.test{background:#fff}
```

The first one is valid CSS and sometimes the result of CSS generators. For example Firebug will generate this line, when background is set to color only.

**Comment by jakubpawlowicz** [comment]
Could you please test if you use the latest version of clean-css? It works fine for me in any of 3.x branches.

```bash
$ echo ".test{background: #ffffff none repeat scroll 0 0;}" | cleancss
.test{background:#fff}
```

**Comment by LeJared** [comment]
If've tested again and found, it only happens when ```--source-map``` option is set.

On Windows i can reproduce this bug with the following line:
```command
type input.css | cleancss --source-map -o output.css
```
where ```input.css``` contains the example CSS

**Comment by jakubpawlowicz** [comment]
It does indeed :+1:

**Comment by LeJared** [comment]
Background-Optimization is still broken in v3.3.9

Example
```css
.test {
  background: url(images/sprite-buttons-2x.png) 0 0 / 100% no-repeat;
}

```
gets optimized to
```css
.test{background:url(images/sprite-buttons-2x.png) no-repeat}
```

expected result
```css
.test{background:url(images/sprite-buttons-2x.png) 0 0/100% no-repeat}
```
Again, it only happens, if ```--source-map``` option is set.

**Comment by jakubpawlowicz** [comment]
I'll take a look at it tomorrow morning. I think we need more source map tests too.

**Comment by jakubpawlowicz** [comment]
It's fixed in 3.3.10 & 3.4.0. Additionally we now run all batch tests with source maps on, and couple additional sourcemap-only issues were already spotted & fixed. See https://github.com/jakubpawlowicz/clean-css/compare/6d3d2e9bfb8bf6cfa2791f65247c58886a66c6ab...2ac0287cd53a910f8df30e8ec4b5d06e3da51c30
